kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -298,6 +342,7 @@ void AddGeometricalUtilitiesToPython(pybind11::module &m)
             const bool is_found = rSelf.FindPointOnMeshSimplified(rCoords,N,p_elem);
             return std::tuple<bool,Vector,Element::Pointer>{is_found,N,p_elem};
         })
+        .def("VectorizedFind", &VectorizedFindHelper<3>)
```

Comment:
```suggestion
        .def("ParallelFind", &VectorizedFindHelper<3>)
```

Makes it more clear IMO

Reply:
Hi @philbucher this is about allowing a vectorized use from python (not necessarily parallel) 
so in my opinion i would keep the name as it is if it is not a blocker to you

Reply:
I was confused why you used this name. you are using indexpartition (which uses omp aka shared memory parallel) and not vectorization in the simd sense

Do you have plans to change the backend?